src/CommunityPatch/Patches/AgriculturePerkPatch.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Linq;
+using System.Reflection;
+using HarmonyLib;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.CampaignSystem.SandBox.GameComponents.Map;
+using TaleWorlds.Core;
+using static CommunityPatch.HarmonyHelpers;
+
+namespace CommunityPatch.Patches {
+
+  sealed class AgriculturePatch : PatchBase<AgriculturePatch> {
+
+    public override bool Applied { get; protected set; }
+
+    private static readonly MethodInfo TargetMethodInfo = typeof(DefaultVillageProductionCalculatorModel).GetMethod("CalculateDailyFoodProductionAmount", BindingFlags.Public | BindingFlags.Instance | BindingFlags.DeclaredOnly);
```

Comment:
Should this not be applied on CalculateDailyProductionAmount instead?

Reply:
You might be right. We did go back and forth a bit on which of the two to patch.

Reply:
> looks like CalculateDailyFoodProductionAmount is actually used to calculate production for food goods in the village in TickFoodProduction in VillageGoodProductionCampaignBehaviour anyways
> so I think CalculateDailyFoodProductionAmount should do the job
> and the result for food production shouldn't be negative, as it just takes the hearthlevel and multiplies it by 0.5 for negative issues

LGTM then